public_website/src/pages/_app.tsx

```diff
@@ -1,15 +1,51 @@
 import React from 'react'
-import type { AppProps } from 'next/app'
+import type { AppContext, AppProps } from 'next/app'
+import App from 'next/app'
+import { Montserrat } from 'next/font/google'
 import { ThemeProvider } from 'styled-components'
 
 import { theme } from '@/theme/theme'
+import { Footer, FooterProps } from '@/ui/components/footer/Footer'
 import GlobalStyles from '@/ui/globalstyles'
+import { fetchCMS } from '@/utils/fetchCMS'
 
-export default function App({ Component, pageProps }: AppProps) {
+const montSerrat = Montserrat({ subsets: ['latin'] })
+
+type MyAppProps = AppProps & { footerData: FooterProps }
+
+export default function MyApp({
+  Component,
+  pageProps,
+  footerData,
+}: MyAppProps) {
   return (
     <ThemeProvider theme={theme}>
       <GlobalStyles />
-      <Component {...pageProps} />
+      {/* Import font globally */}
+      {/* eslint-disable-next-line react/no-unknown-property */}
+      <style jsx global>{`
+        html {
+          font-family: ${montSerrat.style.fontFamily} !important;
+        }
+      `}</style>
```

Comment:
Il faut ajouter exceptionnelleùment un // NOSONAR avec l'explication de pourquoi on ignore 
+ On n'a pas besoin du commentaire sur l'import de la font je pense
```suggestion
      {/* eslint-disable-next-line react/no-unknown-property */}
      <style jsx global>{` // NOSONAR(typescript:S6747) + Explication de pourquoi on ignore
        html {
          font-family: ${montSerrat.style.fontFamily} !important;
        }
      `}</style>
```